.github/workflows/e2e.yaml

```diff
@@ -36,6 +36,9 @@ on:
       - "PROJECT"
       - "sonar-project.properties"
       - "tilt-provider.json"
+concurrency:
```

Comment:
We only added this in our repos, because they were triggered by every push. With the added safety of deployments it's no longer necessary